drop-config/src/lib.rs

```diff
@@ -6,13 +6,14 @@ pub struct Config {
     pub moose: MooseConfig,
 }
 
-#[derive(Debug, Clone, Copy)]
+#[derive(Debug, Clone)]
```

Comment:
If we're dropping `Copy` I suggest wrapping config in `Arc` for cheap cloning in `drop-transfer` crate

Reply:
nice suggestion